src/controller/python/chip/interaction_model/Delegate.cpp

```diff
@@ -79,6 +80,29 @@ CHIP_ERROR PythonInteractionModelDelegate::CommandResponseProcessed(const app::C
     return CHIP_NO_ERROR;
 }
 
+void PythonInteractionModelDelegate::OnReportData(const app::ReadClient * apReadClient, const app::ClusterInfo & aPath,
+                                                  TLV::TLVReader * apData, Protocols::InteractionModel::ProtocolCode status)
+{
+    if (onReportDataFunct != nullptr)
+    {
+        TLV::TLVWriter writer;
+        CHIP_ERROR err = CHIP_NO_ERROR;
+        uint8_t writerBuffer[CHIP_CONFIG_DEFAULT_UDP_MTU_SIZE];
+        writer.Init(writerBuffer, sizeof(writerBuffer));
+        if (apData != nullptr)
+        {
+            TLV::TLVReader tmpReader;
+            tmpReader.Init(*apData);
+            err = writer.CopyElement(TLV::AnonymousTag, tmpReader);
+            ChipLogFunctError(err);
```

Comment:
But press on on failure here?  That doesn't seem right....

Reply:
Added a case for error handling.